<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityWidget</class>
 <widget class="QWidget" name="ActivityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="activityWindow" native="true">
     <property name="maximumSize">
      <size>
       <width>292</width>
       <height>382</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="activityLab1">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="activityLab2">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="activityLab3">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
